app/src/main/java/com/khaithu/a3ctech_android_final_project/presenter/interfacePre/IGetMoviePresenter.kt

```diff
@@ -0,0 +1,8 @@
+package com.khaithu.a3ctech_android_final_project.presenter.interfacePre
+
+import android.view.View
+
+
+interface IGetMoviePresenter {
+    fun getMovie(view: View)
+}
```

Comment:
Name of presenter should same as name of view. 
Example: `HomePresenter`, `HomeFragment`